jupyterhub/handlers/pages.py

```diff
@@ -542,11 +542,50 @@ def sort_key(client):
         oauth_clients = sorted(oauth_clients, key=sort_key, reverse=True)
 
         auth_state = await self.current_user.get_auth_state()
+        expires_in_max = self.settings.get("token_expires_in_max_seconds", 0)
```

Comment:
It can't, really. I've just gotten used to this pattern which was based on porting one Handler to multiple Applications, which isn't relevant to JupyterHub. Should be fine to assume its presence and probably better because a typo here would KeyError.

Reply:
added the assumption that this setting is specified

Reply:
Thanks!